src/app/editor/mod.rs

```diff
@@ -163,6 +163,14 @@ impl Input {
         Ok(AppReturn::Continue)
     }
 
+    fn number_chars_in_current_line(&self) -> usize {
```

Comment:
Can you add `#[allow(dead_code)]` and a comment how this is used for non-ascii characters